src/pretalx/schedule/models/availability.py

```diff
@@ -37,12 +37,16 @@ def serialize(self):
         }
 
     def __str__(self):
-        common = f'start={self.start} end={self.end} event={self.event}'
+        result = f'start={self.start} end={self.end}'
 
+        if hasattr(self, 'event'):
+            result += f' event={self.event}'
```

Comment:
`Availability.event` isn't nullable.

Reply:
It is when we're dealing with unsaved and never-to-be-saved availabilities like the ones generated by `merge_with`. I could make those generate a separate `namedtuple` or something, but that seems like too much overhead and inconsistency.

Reply:
Good point.